MapboxCoreNavigation/RouteController.swift

```diff
@@ -386,6 +388,29 @@ open class RouteController: NSObject {
 
         return CLLocation(coordinate: userCoordinate, altitude: location.altitude, horizontalAccuracy: location.horizontalAccuracy, verticalAccuracy: location.verticalAccuracy, course: userCourse, speed: location.speed, timestamp: location.timestamp)
     }
+    
+    func closestStep(to coordinate: CLLocationCoordinate2D) -> StepIndexDistance? {
```

Comment:
This method can be moved to an extension on RouteLegProgress, because it doesn’t need any of RouteController’s state other than the current leg progress.